imdraw/imdraw.go

```diff
@@ -51,9 +51,11 @@ type IMDraw struct {
 	Precision int
```

Comment:
That makes sense! The reason for the disparity in methods, BTW, is that in a typical-ish use case, the "internal" usages are all going to be <8 points, but "external" usages might be 10k+, so we don't want the externally-used slices to be getting used in the pool, or rotated out to external callers, or we end up with, say, 8 different 10k+ slices.

But you're right about the reuse() thing.